src/plugins/filter/_screen-sm-min.scss

```diff
@@ -1,6 +1,6 @@
 //Screen size medium
 .wb-filter {
 	.input-group {
-		max-width: 80%;
+		max-width: 600px;
```

Comment:
Percentage is problematic in the context of a limited width container (example in a col-md-3, it will take 80% of that column). Like in the "Finder" example added in TagFilter

You might have a more elegant solution but like the rule targeting something in a col-* for example.